src-docs/src/views/guidelines/getting_started/getting_started.js

```diff
@@ -0,0 +1,237 @@
+import React from 'react';
+
+import { EuiCodeBlock, EuiCode, EuiText, EuiSpacer } from '../../../../../src';
+
+import { AppSetup } from './_app_setup';
+import { Tokens } from './_tokens';
+import { CustomizeTokens } from './_customize_tokens';
+
+export const GettingStarted = {
+  title: 'Getting started',
+  sections: [
+    {
+      title: 'Installation',
+      wrapText: false,
+      text: (
+        <>
+          <EuiText grow={false}>
+            <p>
+              EUI is published through{' '}
+              <a href="https://www.npmjs.com/package/@elastic/eui">NPM</a> as a
+              dependency. To install EUI into an existing project, use the{' '}
+              <EuiCode>yarn</EuiCode> CLI (<EuiCode>npm</EuiCode> is not
+              supported).
+            </p>
+          </EuiText>
+          <EuiSpacer />
+          <EuiCodeBlock language="bash" isCopyable fontSize="m">
+            {'yarn add @elastic/eui'}
+          </EuiCodeBlock>
+          <EuiSpacer />
+          <EuiText grow={false}>
+            <p>
+              Note that EUI has{' '}
+              <a href="https://github.com/elastic/eui/package.json">
```

Comment:
This link is failing. I think the right own is the following:

```suggestion
              <a href="https://github.com/elastic/eui/blob/main/package.json">
```